pod/perlthrtut.pod

```diff
@@ -13,7 +13,7 @@ between threads must be explicit.  The user-level interface for I<ithreads>
 uses the L<threads> class.
 
 B<NOTE>: There was another older Perl threading flavor called the 5.005 model
-that used the L<threads> class.  This old model was known to have problems, is
+that used the L<threads> class.  This old model was known to have problems, was
```

Comment:
Not your error, but 5005 threads used the `Thread` class (try `perldoc Thread`), ithreads use the `threads` class.